internal/reconciliations/istio/client.go

```diff
@@ -119,6 +121,17 @@ func (c *IstioClient) Uninstall(ctx context.Context) error {
 	return nil
 }
 
+func installIstioInExternalProcess(mergedIstioOperatorPath string) (string, error) {
+	b, err := exec.Command("./istio_install", mergedIstioOperatorPath).Output()
```

Comment:
Should we have a timeout for this? to not block the reconciliation loop in case of some unexpected issue?